examples/Learning_example/learning_example.ipynb

```diff
@@ -0,0 +1,2295 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    axes.legend()
</pre><p>I think you can add a lot of the output spam (like the &lt;matplotlib.legend.Legend at 0x14ac3ca30&gt; ) by adding ; at the of lines doing plotting etc. that should surpress it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>